### ISSUE TITLE
Can't drag tabs to re-order

### ISSUE THREAD

**Issue by ncalexan** [philikon/VerticalTabs#22]
I'm using Firefox Aurora 17.a2 and have my tab bar displayed on the right.  I can no longer drag-and-drop tabs to re-order them.

**Comment by philikon** [comment]
Yeah they rewrote the tab drag'n'drop code that I monkey-patch. Pull requests accepted :)
